workout.py

```diff
@@ -32,6 +32,13 @@ def generate_workout(time):
     return wo
 
 
+def set_association(workout, exercises):
+    assoc = []
+    for e in exercises:
+        assoc.append(WorkoutExercise(workout=workout, exercise=e))
```

Comment:
minor: could simplify this function to `[WorkoutExercise(workout=workout, exercise=e) for e in exercises]`